vendor/bat-native-ledger/src/bat/ledger/internal/credentials/credentials_promotion.cc

```diff
@@ -332,28 +304,23 @@ void CredentialsPromotion::OnFetchSignedCreds(
     ledger::ResultCallback callback) {
   BLOG(6, ledger::UrlResponseToString(__func__, response));
 
-  if (response.status_code == net::HTTP_ACCEPTED) {
-    callback(ledger::Result::RETRY);
-    return;
-  }
-
-  if (response.status_code != net::HTTP_OK) {
-    callback(ledger::Result::LEDGER_ERROR);
-    return;
-  }
-
   auto get_callback = std::bind(&CredentialsPromotion::SignedCredsSaved,
       this,
       _1,
       trigger,
       callback);
-  common_->GetSignedCredsFromResponse(trigger, response.body, get_callback);
+  common_->GetSignedCredsFromResponse(trigger, response, get_callback);
 }
 
 void CredentialsPromotion::SignedCredsSaved(
     const ledger::Result result,
     const CredentialsTrigger& trigger,
     ledger::ResultCallback callback) {
+  if (result == ledger::Result::RETRY) {
```

Comment:
we need handler for short retry as well

Reply:
I'm not following, `CredentialsCommon::GetSignedCredsFromResponse` only returns `RETRY`. Or are you suggesting a change to the logic?

Reply:
Spoke offline, promotion only supports one type of retry just need to make sure to pass the correct/expected one back.